easybuild/scripts/rpath_wrapper_template.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I'm not sure that calling the original compiler/linker command is going to work when called from within a Python shell, I suspect that may lead to problems.

Total overhead of this bash+Python wrapper approach is 10-15%, which is very acceptable imho, you'll barely notice.